io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -99,14 +99,14 @@ public function channelReadXml(ReadableChannel readableChannel) returns @tainted
     }
 }
 
-# Write a string content to a channel.
+# Write a string content to a channel
 # ```ballerina
 # string content = "Hello Universe..!!";
 # io:Error result = io:channelWriteString(writableChannel, content);
 # ```
-# + writableChannel - A writable channel
-# + content - String content to read
-# + return - Either `io:Error` or `()`
+# + writableChannel - A writable channel. The possible inputs are `io:WritableByteChannel` or `io:WritableCharacterChannel`.
+# + content - String content to write
```

Comment:
```suggestion
# + content - String content to be written
```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8